webapp/packages/core-authentication/src/AuthProvidersResource.ts

```diff
@@ -20,19 +20,19 @@ import { ServerConfigResource } from '@cloudbeaver/core-root';
 import { AuthProviderConfigurationInfoFragment, AuthProviderInfoFragment, GraphQLService } from '@cloudbeaver/core-sdk';
 
 import { AuthConfigurationsResource } from './AuthConfigurationsResource';
-import { AuthSettingsService } from './AuthSettingsService';
 
 export type AuthProvider = NonNullable<AuthProviderInfoFragment>;
 export type AuthProviderConfiguration = NonNullable<AuthProviderConfigurationInfoFragment>;
 
 @injectable()
 export class AuthProvidersResource extends CachedMapResource<string, AuthProvider> {
   get configurable(): AuthProvider[] {
-    return this.values.filter(provider => provider.configurable);
+    const enabledAuthProviders = new Set(this.serverConfigResource.enabledAuthProviders);
```

Comment:
We dont want this logic in resource, please move it to the plugin-identity-providers-administration and filter on the component level, also we want to show some placeholder (we need co thing about the ui) if there is no available providers